docs/swiftui/samples/accessibility-action/index.mdx

```diff
@@ -0,0 +1,18 @@
+---
+locale: en
+slug: /swiftui/samples/accessibility-action
+contentful_id: aDLlvHRKDdYmyFQcgRvUG
+title: Accessibility action in SwiftUI
+description: Accessibility actions provide alternative ways for users of assistive technologies to perform actions.
+keywords: [iOS, SwiftUI, accessibility, code sample]
+authors: [janjaap]
+date: 2023-01-23
```

Comment:
Set date to `2024-07-09` for all SwiftUI pages (you can use VSCode regex to bulk replace all dates`

Reply:
Search: `date: [0-9-]+ (enable regex)`

Replace: `date: 2024-07-09`

Files to include: `i18n/nl/docusaurus-plugin-content-docs/current/{PLATFORM}/*`